src/mds/config.py

```diff
@@ -68,9 +72,3 @@ def __init__(self, value):
 DATA_ACCESS_SERVICE_ENDPOINT = config(
     "DATA_ACCESS_SERVICE_ENDPOINT", cast=str, default="http://fence-service"
 )
-
-if TESTING:
-    DB_DATABASE = "test_" + (DB_DATABASE or "mds")
-    TEST_KEEP_DB = config("TEST_KEEP_DB", cast=bool, default=False)
-    INDEXING_SERVICE_ENDPOINT = "http://localhost"
```

Comment:
where did these go?

Reply:
I thought that because requests to `INDEXING_SERVICE_ENDPOINT` and `DATA_ACCESS_SERVICE_ENDPOINT` are mocked in the tests, maybe the default values are acceptable?